File: 해시/전화번호 목록.js
function solution(phone_book) {
    var answer = true;
    phone_book.sort()
    for(let i =0;i<phone_book.length-1;i++){
        let isSame = true;
        for(let j=0;j<phone_book[i].length;j++){
            if(phone_book[i][j] !== phone_book[i+1][j]){
                isSame=false
                break
            }
        }
        if(isSame){
            answer=false
            break
        }
    }
    return answer;
}

File: 해시/전화번호 목록.py
def solution(phone_book):
    answer = True
    phone_book.sort()
    
    for i in range(0, len(phone_book)-1):
        flag = True
        for j in range(len(phone_book[i])):
            if phone_book[i+1][j]!=phone_book[i][j]:
                flag=False
                break
        if flag :
            answer =False
            break
    return answer
